it/uk/gov/hmrc/agentsubscriptionfrontend/support/Css.scala

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmrc.agentsubscriptionfrontend.support
+
+
+object Css {
+  val ERROR_SUMMARY_TITLE = "#error-summary-title"
+  val ERROR_SUMMARY_LINK = ".govuk-list.govuk-error-summary__list li a"
+  val errorSummaryLinkWithHref  = (href: String)  => s".govuk-list.govuk-error-summary__list li a[href=$href]"
+  val H1 = "main h1"
+  val PRE_H1 = "main .govuk-caption-l"
+  val paragraphs = "main p"
+  def errorSummaryForField(id: String): String ={
+    s".govuk-error-summary__body li a[href=#${id}]"
+  }
+  def errorForField(id: String): String = s"span#${id}-error"
```

Comment:
and this css selector  can be used to test that the error summary link text for a given field of a given id is there where it should be.

Reply:
Try now Caroline